packages/react-instantsearch-core/src/components/Index.js

```diff
@@ -1,7 +1,8 @@
+/* eslint valid-jsdoc: 0 */
```

Comment:
I read this rule is deprecated so I guess it can be disabled globally?
I'm not sure what replaces it